### DISCUSSION TITLE
Sync Full node - wallet - farm node

### DISCUSSION THREAD

**Discussion by casa10** [Chia-Network/chia-blockchain#2548] [General]
Hello,
i use chia 1.1.1.
I have these screen shot about full node status, wallet status, farm node status. 
It's already more than 30 hours i wait and it's not finished.
1) is someone can explain me please why it's so long ? 
2) is it possible to win time to be operational ?
3) if i shut down chia gui and restart, do i need still so long time to synchronize all ?
Thank you to the community
![Capture](https://user-images.githubusercontent.com/8081413/115949914-66d41b80-a4d8-11eb-9280-79f86123f433.JPG)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
